validation/validation_service.py

```diff
@@ -0,0 +1,70 @@
+import requests
+import json
+import os
+
+from .not_known_entity_exception import NotKnownEntityException
+from .validation_state import ValidationState
+
+
+class ValidationService:
+    SCHEMA_FILENAME_PREFIX = "covid_data_uploader-"
```

Comment:
why this giant prefix?

Reply:
We probably don't need it and we could rename the files. It was only a left over from the starting point.